docs/_articles/en/v3.x/release-information.md

```diff
@@ -5,775 +5,47 @@ lang: en
 
 ### To update the plugin, please follow the linked [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in)
 
-## [v2.13.1](https://github.com/forcedotcom/sfdx-scanner/tree/v2.13.1) (03-23-2022)
+## [v3.0.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.0.0) (04-26-2022)
 
-[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.13.0...v2.13.1)
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.13.1...v3.0.0)
 
 ### Release Summary
 
-* Updated PMD to 6.43.0
-* Upgrades to local RetireJS Vulnerability Repository
+* Updated PMD to 6.44.0
+* Updated Eslint to 6.8.0
+* Updated RetireJS to 2.2.5
+* Updated Typescript to 4.6.2
+* Fixed linting issues related to upgrades
```

Comment:
Do we have to callout RetireJS is enabled by default? here?

Reply:
Good one 👍